JustBrewIt/BreweryAPI.cs

```diff
@@ -19,7 +19,7 @@
 //
 //    var welcome = Welcome.FromJson(jsonString);
 
-namespace JustBrewItQuickType
+namespace JustBrewIt
```

Comment:
Suggestion 5:  Model pages have default namespace and class name.   Meaningful naming practice improves code readability.
•	 
Action: I have changed the namespace and class naming in your models.  
Reference: 
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/namespaces/using-namespaces